libs/react-hooks/src/use-small-screen/use-small-screen.ts

```diff
@@ -3,7 +3,7 @@ import { useState, useEffect, useCallback } from 'react'
 const DEFAULT_BREAKPOINT = 768
 
 export function useSmallScreen(width = DEFAULT_BREAKPOINT) {
-  const [isSmallScreen, setIsSmallScreen] = useState<boolean>(false)
+  const [isSmallScreen, setIsSmallScreen] = useState<boolean>(true)
```

Comment:
Isn't this a breaking change?

Reply:
I checked where it was used and looked like `true` made more sense as the initial value in all cases. In some places it makes no difference but in others it avoids an unnecessary layout shift. We can review it together.